src/components/LoansByCategory/CategoryRow.vue

```diff
@@ -178,7 +178,8 @@ export default {
 	methods: {
 		saveWindowWidth() {
 			this.windowWidth = window.innerWidth;
-			this.wrapperWidth = this.$refs.innerWrapper.clientWidth;
+			// TODO: New Countries for You code is getting executed even for NON Logged in lenders (no loans, no width)
+			this.wrapperWidth = this.$refs.innerWrapper ? this.$refs.innerWrapper.clientWidth : 0;
```

Comment:
Waiting on future plans for New countries category row... so just introducing a check until that plan is finalized.